### ISSUE TITLE
Is possible to extend for other language, like portuguese brazil?

### ISSUE THREAD

**Issue by michel-fernandes** [deepconvolution/LipNet#8]
I'm a post covid patient and because of one of the consequences I can't talk. So I see the application as an alternative for accessibility and communication, for example using video conferencing. Is there a possibility for me to work on this need?

**Comment by Saqhas** [comment]
Yes you can work on it. All the code are on the repo. If you need any help in understanding the model or the dataset then please feel free to contact.
